Fix:

error: call to undeclared function 'xmlCleanupParser'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]

https://sourceforge.net/p/gnucobol/code/5295/
--- libcob/common.c.orig	2023-07-28 12:16:38.000000000 -0500
+++ libcob/common.c	2024-09-20 21:38:22.000000000 -0500
@@ -134,6 +134,7 @@
 #endif
 
 #if defined (WITH_XML2)
+#include <libxml/parser.h>
 #include <libxml/xmlversion.h>
 #include <libxml/xmlwriter.h>
 #endif
